build_data.js

```diff
@@ -263,7 +263,6 @@ function suggestionsToPresets(presets) {
             geometry: preset.geometry,
             tags: Object.assign({}, preset.tags, wikidataTag),
             addTags: suggestion.tags,
-            removeTags: suggestion.tags,
```

Comment:
This change will btw somewhat reduce the size of the `presets.json`

Reply:
Yeah, tbh for iD v3 we should find a way to bundle the suggestions separately anyway.  They are huge.